tests/utils.py

```diff
@@ -186,3 +187,26 @@ def check_synapse(syn, edges, selection, **kwargs):
 
     if _get_attr("n_rrp_vesicles", kwargs, edges, selection, syn_id) >= 0:
         assert syn.Nrrp == _get_attr("n_rrp_vesicles", kwargs, edges, selection, syn_id)
+
+
+def check_signal_peaks(x, ref_peaks_pos, threshold=0.1):
+    """
+    Check the given signal peaks comparing with the given
+    reference
+
+    Args:
+        x: given signal, typically voltage.
+        ref_peaks_pos: the position of the signal peaks
+        taken as reference.
+        threshold: peak detection threshold with values
+        between 0 and 1, measured over the normalized
+        signal.
+
+    Raises:
+        AssertionError: If any of the reference peak
+        positions doesn't match with the obtained peaks
+    """
+    x_norm = (x - x.min()) / (x.max() - x.min())
```

Comment:
using prominence is usually good but I am concerned about this norm. May you try the test where you combine a spiking signal with a steady rising one? 

y = spike_signal + a * t

where a is a fixed coefficient. It may happen if we are measuring accumulations of ions or even voltage or current